ocaml/xapi/xapi_globs.ml

```diff
@@ -911,12 +911,14 @@ let ntlm_auth_cmd = ref "/usr/bin/ntlm_auth"
 
 let winbind_debug_level = ref 2
 
-let winbind_cache_time = ref 300
+let winbind_cache_time = ref 10
```

Comment:
what's the benefit of having a low cache time here? I would expect the data on the windows server to be changing pretty infrequently

docs for this config setting: https://www.samba.org/samba/docs/current/man-html/smb.conf.5.html#WINBINDCACHETIME

Reply:
See the updated design page.
xapi query such info from cache, like query sid from cache, and query user information from DC by sid, 
if the user infomate is updated ( delete and re-add back), then xapi cannot resolve the SID.

This is the time how often winbind refresh its cache.  and the load looks okay,